pkg/controller/controller_utils_test.go

```diff
@@ -125,10 +125,29 @@ func TestTiKVCapacity(t *testing.T) {
 		{
 			name: "100GiB",
 			limit: &v1alpha1.ResourceRequirement{
-				Storage: "100Gi",
+				Storage: "100GiB",
```

Comment:
`Storage: "100Gi"` has already been tested in previous test case. I guess this should be `100GiB`.